python_transport/wirepas_gateway/transport_service.py

```diff
@@ -458,6 +458,9 @@ def _on_send_data_cmd_received(self, client, userdata, message):
             self.logger.error(str(e))
             return
 
+        if not self.msg_manager.add_msg(request.req_id):
```

Comment:
a comment here would be welcome to add a brief explanation around the logic at play